plutus-tx-plugin/test/StdLib/Spec.hs

```diff
@@ -76,8 +82,8 @@ testQuotRem :: Property
 testQuotRem = Hedgehog.property $ do
     let gen = Gen.integral (Range.linear (-10000) 100000)
     (n1, n2) <- Hedgehog.forAll $ (,) <$> gen <*> gen
-    let ghcResult = quotRem n1 n2
-        plutusResult = Ratio.quotRem n1 n2
+    ghcResult <- tryHard $ quotRem n1 n2
+    plutusResult <- tryHard $ Ratio.quotRem n1 n2
```

Comment:
How does `Ratio.quotRem` fail? Is it an exception? I thought we'd get some kind of pure `EvaluationFailure` or something in Plutus?

Reply:
Looking at it, this test is weird. It tests the *Haskell* version of the builtin, which delegates to the Haskell function that implements the semantics, but it's a little weird. We should really test evaluating a PLC program that uses the builtin instead!

Not sure whether to fix this now, I really just wanted to make it no longer non-deterministically fail. I'll see if I have some more time to tweak it.

Reply:
Right, I thought it was testing Plutus somehow.

Reply:
> How does `Ratio.quotRem` fail? Is it an exception? I thought we'd get some kind of pure `EvaluationFailure` or something in Plutus?

I think that'll give you a "Divide by zero" exception, just like `div` does in Haskell, or `divideInteger` in PLC (and the same with `quot`, `rem`, `div`, `mod` and their PLC versions).